dll/directx/wine/d3dx9_36/d3dx9_36.spec

```diff
@@ -269,7 +269,7 @@
 @ stdcall -stub D3DXSHMultiply6(ptr ptr ptr)
 @ stdcall -stub D3DXSHPRTCompSplitMeshSC(ptr long long ptr long ptr long long ptr ptr long ptr ptr ptr ptr ptr)
 @ stdcall -stub D3DXSHPRTCompSuperCluster(ptr ptr long long ptr ptr)
-@ stdcall -stub D3DXSHProjectCubeMap(long ptr ptr ptr ptr)
```

Comment:
A Wine sync seems to be appropriate for these spec files -- however in many modules, our spec files are forked. Should the sync script take this into account somehow (and simply alert when Wine's .spec changes, as for Makefile.in)?

Reply:
Problem here is that spec2def doesn't accept "@ stub" entries.

Also, if a patch hunk fails to apply, the script stops and lets you amend the commit, and you can proceed further with the update process. The problem with forked file is true for any files, not only .spec files.